contents/articles/2016-11_facevr/index.md

```diff
@@ -0,0 +1,26 @@
+--- 
+title: "FaceVR: Real-Time Facial Reenactment and Eye Gaze Control in VR"
+date: Thu Nov 17 2016 10:49:16 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "A novel approach for real-time real-time gaze-aware facial performance capture system to drive the photo-realistic reconstructed digital face in virtual reality..."
+featured_img: "../../../../media/images/blog/face_vrb.png"
+featured_img_thumb: "../../../../mesdia/images/blog/thumb/face_vr.png"
+---
+
+<img class="Feature-image u-padBottom--1" src="../../../../media/images/blog/face_vr.png" alt="Real-Time Facial Reenactment and Eye Gaze Control in VR">
+
+Source: [FaceVR: Real-Time Facial Reenactment and Eye Gaze Control in VR](https://arxiv.org/pdf/1610.03151.pdf)
+
+<br>
+
+[Justus Thies](http://lgdv.cs.fau.de/people/card/justus/thies/) et al., presents a novel approach for real-time gaze-aware facial capture system to drive the photo-realistic reconstructed digital face in virtual reality. A facial reenactment that can transfer facial expressions and realistic eye appearance between a source and a target actor video.
```

Comment:
change "presents a novel approach" to "have developed a novel approach..."

change "A facial reenactmaent..." to "Their approach enables facial reenactment that can transfer..."